src/advanced-slots.tsx

```diff
@@ -0,0 +1,350 @@
+        </div>
+    );
+};
+
+interface RowBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const RowBlock: React.FC<RowBlockProps> = ({
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+            }}
+            aria-label={ariaLabel}
+        >
+            {(title || stackingGroup || description) && (
+                <>
+                    {stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular>{title}</Text2>
+                                </div>
+                            )}
+                            <div style={{zIndex: '0'}}>{stackingGroup}</div>
+                        </div>
+                    )}
+                    {!stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular as="div">
+                                        {title}
+                                    </Text2>
+                                </div>
+                            )}
+                            <div>
+                                <Text2 regular color={vars.colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </div>
+                        </div>
+                    )}
+                </>
+            )}
+        </div>
+    );
+};
+
+interface SimpleBlockProps {
+    image?: RendersNullableElement<typeof Image>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const SimpleBlock: React.FC<SimpleBlockProps> = ({image, description, 'aria-label': ariaLabel}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            <div
+                className={sprinkles({
+                    display: 'flex',
+                    alignItems: 'center',
+                    height: 40,
```

Comment:
I think that the component just has to hug the content. So, I think that you can remove this height

Reply:
But in the design specifications it has a fixed size, both with or without an image

Reply:
we can never have fixed heights because height depends on the text content, and font size can be bigger (for accesibility reasons) if the user has change their phone settings (you can use storybook font size control to simulate this).